Unable to build Docker Native image "manifest unknown"

### Issue description

Hi, I am not able to build the Docker native image using **micronaut** version `3.3.0`

I've tried **Gitter**, but I didn't get any help.

Here is the error
```
$./mvnw clean package -Pgraalvm -Dpackaging=docker-native
[INFO] Scanning for projects...
[INFO]
[INFO] --------------------------< com.example:demo >--------------------------
[INFO] Building demo 0.1
[INFO] ---------------------------[ docker-native ]----------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ demo ---
[INFO] Deleting /demo/target
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:resources (default-resources) @ demo ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] Copying 2 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ demo ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 1 source file to /demo/target/classes
[INFO]
[INFO] --- micronaut-maven-plugin:3.1.0:graalvm-resources (default-graalvm-resources) @ demo ---
[INFO] Generating /demo/target/classes/META-INF/native-image/com.example/demo/resource-config.json
[INFO]
[INFO] --- micronaut-maven-plugin:3.1.0:docker-native (default-docker-native) @ demo ---
[INFO] Exposing port: 8080
[INFO] Using BASE_IMAGE: ghcr.io/graalvm/native-image:java11-22.0.0.2
[INFO] Using BASE_IMAGE_RUN: frolvlad/alpine-glibc:alpine-3.12
[INFO] Using CLASS_NAME: com.example.Application
[INFO] Step 1/16 : ARG BASE_IMAGE
[INFO]
[INFO] Step 2/16 : ARG BASE_IMAGE_RUN
[INFO]
[INFO] Step 3/16 : FROM ${BASE_IMAGE} AS builder
[INFO]
[ERROR] manifest unknown
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.199 s
[INFO] Finished at: 2022-02-04T11:17:23+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.micronaut.build:micronaut-maven-plugin:3.1.0:docker-native (default-docker-native) on project demo: Execution default-docker-native of goal io.micronaut.build:micronaut-maven-plugin:3.1.0:docker-native failed: Could not build image: manifest unknown -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

**Steps to reproduce**
- `$ curl --location --request GET 'https://launch.micronaut.io/create/default/com.example.demo?lang=JAVA&build=MAVEN&test=JUNIT&javaVersion=JDK_11&features=graalvm' --output demo.zip`
- unzip the `demo.zip` file
- `$ cd demo`
- `$ ./mvnw clean package -Pgraalvm -Dpackaging=docker-native`

Thanks!

--- a/src/it/dockerfile-docker-native-distroless/Dockerfile
+++ b/src/it/dockerfile-docker-native-distroless/Dockerfile
@@ -1,4 +1,4 @@
-FROM ghcr.io/graalvm/native-image:java11-21.3.0 AS builder
+FROM ghcr.io/graalvm/native-image:ol8-java11-21.3.1 AS builder
 WORKDIR /home/app
 
 COPY classes /home/app/classes

--- a/src/it/dockerfile-docker-native-distroless/pom.xml
+++ b/src/it/dockerfile-docker-native-distroless/pom.xml
@@ -20,7 +20,7 @@
     <exec.mainClass>io.micronaut.build.examples.Application</exec.mainClass>
     <packaging>jar</packaging>
     <micronaut.runtime>netty</micronaut.runtime>
-    <graal.version>21.3.0</graal.version>
+    <graal.version>21.3.1</graal.version>
   </properties>
 
   <repositories>

--- a/src/it/dockerfile-docker-native-lambda/Dockerfile
+++ b/src/it/dockerfile-docker-native-lambda/Dockerfile
@@ -1,9 +1,9 @@
 FROM amazonlinux:2 AS graalvm
 ENV LANG=en_US.UTF-8
 RUN yum update -y && yum install -y gcc gcc-c++ zlib-devel zip tar gzip && yum clean all
-RUN curl -4 -L https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-21.3.0/graalvm-ce-java11-linux-amd64-21.3.0.tar.gz -o /tmp/graalvm.tar.gz \
+RUN curl -4 -L https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-21.3.1/graalvm-ce-java11-linux-amd64-21.3.1.tar.gz -o /tmp/graalvm.tar.gz \
     && tar -zxf /tmp/graalvm.tar.gz -C /tmp \
-    && mv /tmp/graalvm-ce-java11-21.3.0 /usr/lib/graalvm \
+    && mv /tmp/graalvm-ce-java11-21.3.1 /usr/lib/graalvm \
     && rm -rf /tmp/*
 RUN /usr/lib/graalvm/bin/gu install native-image
 ENV PATH=/usr/lib/graalvm/bin:${PATH}

--- a/src/it/dockerfile-docker-native-lambda/pom.xml
+++ b/src/it/dockerfile-docker-native-lambda/pom.xml
@@ -19,7 +19,7 @@
     <micronaut-maven-plugin.version>@project.version@</micronaut-maven-plugin.version>
     <exec.mainClass>io.micronaut.function.aws.runtime.MicronautLambdaRuntime</exec.mainClass>
     <micronaut.runtime>lambda</micronaut.runtime>
-    <graal.version>21.3.0</graal.version>
+    <graal.version>21.3.1</graal.version>
   </properties>
 
   <repositories>

--- a/src/it/dockerfile-docker-native-netty/Dockerfile
+++ b/src/it/dockerfile-docker-native-netty/Dockerfile
@@ -1,4 +1,4 @@
-FROM ghcr.io/graalvm/native-image:java11-21.3.0 AS builder
+FROM ghcr.io/graalvm/native-image:ol8-java11-21.3.1 AS builder
 WORKDIR /home/app
 
 COPY classes /home/app/classes

--- a/src/it/dockerfile-docker-native-netty/pom.xml
+++ b/src/it/dockerfile-docker-native-netty/pom.xml
@@ -20,7 +20,7 @@
     <exec.mainClass>io.micronaut.build.examples.Application</exec.mainClass>
     <packaging>jar</packaging>
     <micronaut.runtime>netty</micronaut.runtime>
-    <graal.version>21.3.0</graal.version>
+    <graal.version>21.3.1</graal.version>
   </properties>
 
   <repositories>

--- a/src/it/dockerfile-docker-native-oracle-function/Dockerfile
+++ b/src/it/dockerfile-docker-native-oracle-function/Dockerfile
@@ -1,4 +1,4 @@
-FROM ghcr.io/graalvm/native-image:java11-21.3.0 AS builder
+FROM ghcr.io/graalvm/native-image:ol8-java11-21.3.1 AS builder
 WORKDIR /home/app
 
 COPY classes /home/app/classes

--- a/src/it/dockerfile-docker-native-oracle-function/pom.xml
+++ b/src/it/dockerfile-docker-native-oracle-function/pom.xml
@@ -20,7 +20,7 @@
     <exec.mainClass>io.micronaut.build.examples.Application</exec.mainClass>
     <packaging>jar</packaging>
     <micronaut.runtime>netty</micronaut.runtime>
-    <graal.version>21.3.0</graal.version>
+    <graal.version>21.3.1</graal.version>
   </properties>
 
   <repositories>

--- a/src/it/dockerfile-docker-native-static/Dockerfile
+++ b/src/it/dockerfile-docker-native-static/Dockerfile
@@ -1,4 +1,4 @@
-FROM ghcr.io/graalvm/graalvm-ce:java11-21.3.0 AS builder
+FROM ghcr.io/graalvm/graalvm-ce:ol8-java11-21.3.1 AS builder
 RUN gu install native-image
 
 RUN curl -L -o musl.tar.gz https://more.musl.cc/10/x86_64-linux-musl/x86_64-linux-musl-native.tgz && \

--- a/src/it/dockerfile-docker-native-static/pom.xml
+++ b/src/it/dockerfile-docker-native-static/pom.xml
@@ -20,7 +20,7 @@
     <exec.mainClass>io.micronaut.build.examples.Application</exec.mainClass>
     <packaging>jar</packaging>
     <micronaut.runtime>netty</micronaut.runtime>
-    <graal.version>21.3.0</graal.version>
+    <graal.version>21.3.1</graal.version>
   </properties>
 
   <repositories>

--- a/src/it/package-docker-native-distroless/pom.xml
+++ b/src/it/package-docker-native-distroless/pom.xml
@@ -20,7 +20,7 @@
     <exec.mainClass>io.micronaut.build.examples.Application</exec.mainClass>
     <packaging>jar</packaging>
     <micronaut.runtime>netty</micronaut.runtime>
-    <graal.version>21.3.0</graal.version>
+    <graal.version>21.3.1</graal.version>
   </properties>
 
   <repositories>

--- a/src/it/package-docker-native-static/Dockerfile
+++ b/src/it/package-docker-native-static/Dockerfile
@@ -1,4 +1,4 @@
-FROM ghcr.io/graalvm/graalvm-ce:java11-21.3.0 AS builder
+FROM ghcr.io/graalvm/graalvm-ce:ol8-java11-21.3.1 AS builder
 RUN gu install native-image
 WORKDIR /home/app
 

--- a/src/it/package-docker-native-static/pom.xml
+++ b/src/it/package-docker-native-static/pom.xml
@@ -20,7 +20,7 @@
     <exec.mainClass>io.micronaut.build.examples.Application</exec.mainClass>
     <packaging>jar</packaging>
     <micronaut.runtime>netty</micronaut.runtime>
-    <graal.version>21.3.0</graal.version>
+    <graal.version>21.3.1</graal.version>
   </properties>
 
   <repositories>

--- a/src/it/package-docker-native/pom.xml
+++ b/src/it/package-docker-native/pom.xml
@@ -20,7 +20,7 @@
     <exec.mainClass>io.micronaut.build.examples.Application</exec.mainClass>
     <packaging>jar</packaging>
     <micronaut.runtime>netty</micronaut.runtime>
-    <graal.version>21.3.0</graal.version>
+    <graal.version>21.3.1</graal.version>
   </properties>
 
   <repositories>

--- a/src/it/package-native-image/pom.xml
+++ b/src/it/package-native-image/pom.xml
@@ -20,7 +20,7 @@
     <exec.mainClass>io.micronaut.build.examples.Application</exec.mainClass>
     <packaging>jar</packaging>
     <micronaut.runtime>netty</micronaut.runtime>
-    <graal.version>21.3.0</graal.version>
+    <graal.version>21.3.1</graal.version>
   </properties>
 
   <repositories>

--- a/src/main/java/io/micronaut/build/AbstractDockerMojo.java
+++ b/src/main/java/io/micronaut/build/AbstractDockerMojo.java
@@ -111,9 +111,9 @@ public abstract class AbstractDockerMojo extends AbstractMojo {
     protected String getFrom() {
         if (staticNativeImage) {
             // For building a static native image we need a base image with tools (cc, make,...) already installed
-            return jibConfigurationService.getFromImage().orElse("ghcr.io/graalvm/graalvm-ce:" + graalVmJvmVersion() + "-" + graalVmVersion());
+            return jibConfigurationService.getFromImage().orElse("ghcr.io/graalvm/graalvm-ce:ol8-" + graalVmJvmVersion() + "-" + graalVmVersion());
         } else {
-            return jibConfigurationService.getFromImage().orElse("ghcr.io/graalvm/native-image:" + graalVmJvmVersion() + "-" + graalVmVersion());
+            return jibConfigurationService.getFromImage().orElse("ghcr.io/graalvm/native-image:ol8-" + graalVmJvmVersion() + "-" + graalVmVersion());
         }
     }
 

